contracts.json

```diff
@@ -93,16 +93,16 @@
   },
   "mumbai": {
     "BridgedPolygonNORI": {
-      "proxyAddress": "0x6E570A15C0C39b5F29388DEEa6ebda8CDDc40587"
+      "proxyAddress": "0x8D7713C76A2A6Ec517FB3C032524dd86425E24f4"
     },
     "Removal": {
-      "proxyAddress": "0xa484c59eB391afFd88C7de0dBF4B6E7e2A9E84fE"
+      "proxyAddress": "0xa051E9EeaC803d2fCA1DbF415b78AD9BfEB723b0"
     },
     "Certificate": {
       "proxyAddress": "0xF5bafc8B26Fb38ea788Bc6eAf3d84ba3bc4Ca91C"
     },
     "Market": {
-      "proxyAddress": "0x61819E8238A3493547Ae1131dc657D87F62833FD"
+      "proxyAddress": "0x53D76F0d411F8Fc5278999Ae7929C163Dbeac7d7"
```

Comment:
for some reason the addresses here do not match the addresses of these contracts as registered in Defender... so something may still be not right about our deployment process because we want to keep these in sync.

The changes in this PR are manually changing these addresses to point to the defender addresses so that our hardhat task commands would work on the correct contracts.... but this is still a mystery we need to solve